src/SonarQube.Client/Models/Connections.cs

```diff
@@ -20,17 +20,27 @@
 
 using System;
 using System.Security;
+using SonarQube.Client.Helpers;
 
-namespace SonarLint.VisualStudio.Integration.Service
+namespace SonarQube.Client.Models
 {
+    // TODO: I want to use this class but there are a lot changes to do so keep using the other now on.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)